app/models/zoom_meeting.rb

```diff
@@ -8,7 +8,8 @@ def self.from_meeting_details(meeting_url)
     raise invalid_error if meeting_details[:code] == 3001
     raise no_meeting_error if meeting_details[:code] == 2300
     raise personal_meeting_error if meeting_details[:start_time].nil?
-    
+    raise no_participants_error if meeting_details[:participants_count] == 0
```

Comment:
raising the error looks good, but you'll need another way to check if there are zero participants. See my comment about the fixture.

Reply:
Removed the raise error and added a `transaction` to the CreateAttendanceFacade since an error was already raised in the `take_participant_attendance` for no participants.

Reply:
One question we did have:  For some reason whenever we added the transaction to the CreateAttendanceFacade it broke all the successful create attendances since the attendance variable on line 6 (of the controller) returned an unsaved attendance record.  However whenever we changed the `take_attendance` method in the CreateAttendanceFacade to implicitly return the attendance record it persisted to the controller. 
Not sure why changing an explicit return to an implicit one would return the saved record.